app/code/Magento/GraphQl/Controller/GraphQl.php

```diff
@@ -88,6 +90,11 @@ class GraphQl implements FrontControllerInterface
      * @var ContextFactoryInterface
      */
     private $contextFactory;
+    
+    /** 
+     * @var AreaList $areaList
+     */
+    private $appState;
```

Comment:
I think `private $appState;` should be change into `private $areaList;`

Reply:
Fixed. Stupid me. I had an error here before but only half fixed it.